.eslintignore

```diff
@@ -0,0 +1,3 @@
+tmp
+lib
+dist
```

Comment:
the dist folder that the bundle will be stored in needs to be committed to git, otherwise the action won't work (basically what is pointed to in the action.yml as `main` file)

Reply:
Yeah, it is using `lib/main.js`: https://github.com/ExodusMovement/fork-sync/blob/8978967b7d0041a26270976300f9892c99000ed3/action.yml#L60

Reply:
I removed this from `eslintignore` to avoid any confusion:

https://github.com/ExodusMovement/fork-sync/pull/4/commits/8792ea24419d680ddc48799757bfd5921e4564f0

Reply:
`lib` is still ignored though, the build output needs to be committed (or rather the bundle output after changing this to use ncc)

Reply:
Humm, maybe you are confusing this with `gitignore`? This is simply ignoring it when linting as we don't wanna lint the output, right?

Reply:
ups, haha sorry. you're right that's what I've been doing. Today is not my day

Reply:
Lol, I do stuff like that everyday, then no day is my day 😆